<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>domain9</class>
 <widget class="QDialog" name="domain9">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>441</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>preamble: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEditPreamble"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>id : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEditId"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>type: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEditType"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>A2 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEditA2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>A1 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEditA1"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>A0 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lEditA0"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnConfirm9">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
